crates/prover/src/runner.rs

```diff
@@ -474,7 +474,7 @@ where
             sequencer_commitments.sort_unstable();
 
             let (sequencer_commitments, preproven_commitments) =
-                self.filter_out_proven_commitments(&sequencer_commitments)?;
+                filter_out_finalized_commitments(&self.ledger_db, &sequencer_commitments)?;
```

Comment:
this should have stayed filter out proven

Reply:
and on line 831 the prover should mark the ranges as proven, since it knows it already proved them.

Reply:
a mistake we did on not proving preproven_commitments pr :D